caput/misc.py

```diff
@@ -196,3 +199,77 @@ def tmpfile(self):
     @property
     def lockfile(self):
         return self.tmpfile + ".lock"
+
+
+# Wrapper for getfullargspec
+def getfullargspec(f):
+    """Python 2 compatible implementation of `inspect.getfullargspec`.
+
+    Parameters
+    ----------
+    f : function
+        Callable to inspect.
+
+    Returns
+    -------
+    fullargspec : namedtuple
+        Named tuple with various fields. See `inspect.getfullargspec`.
+    """
+
+    import inspect
```

Comment:
Is there a good reason this isn't ontop? https://www.python.org/dev/peps/pep-0008/#imports